src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -14,6 +14,15 @@
   <PropertyGroup Condition=" '$(TargetsBrowser)' == 'true'">
     <DefineConstants>$(DefineConstants);TARGETS_BROWSER</DefineConstants>
   </PropertyGroup>
+  <!-- ILLinker settings -->
```

Comment:
Do we have same docs how this linker stuff works? Could you point me to it?
I'd like to understand these changes better.

Reply:
@eerhardt wrote the build task but I'm not sure there is any doc. You can check the code at https://github.com/dotnet/runtime/blob/master/eng/illink.targets

Reply:
A good starting overview is: https://github.com/dotnet/designs/blob/master/accepted/2020/linking-libraries.md

The feature switch design is good at explaining how certain code can be trimmed when a user doesn't want it: https://github.com/dotnet/designs/blob/master/accepted/2020/feature-switch.md.